src/sdk/namespace_fs.js

```diff
@@ -299,6 +299,26 @@ class NamespaceFS {
      * @param {ListParams} params 
      */
     async list_objects(params, object_sdk) {
+        return this._list_objects(params, object_sdk, false);
+    }
+
+    async list_object_versions(params, object_sdk) {
+        return this._list_objects(params, object_sdk, true);
+    }
+
+    async _list_version_dir_objects(results, entry_name, dir_key) {
+        const filePath = path.normalize(path.join(this.bucket_path, '/', dir_key, entry_name));
+        const versions = await readdir(filePath);
```

Comment:
you should use await nb_native().fs.readdir(fs_context, filePath);

Reply:
sure